sidecar/src/service/tracing/mod.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright 2021-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+
+pub(crate) use trace_flusher::TraceFlusher;
+
+use trace_send_data::TraceSendData;
```

Comment:
What's the point of this use?

Reply:
it elevates `TraceSendData` to the top level of the module so you can import it as `tracing::TraceSendData` instead of `tracing::trace_send_data::TraceSendData`. 

There are situations where you want the full namespace path, but I think `TraceSendData` is an example where there is no benefit.

Reply:
I was mostly referring to the fact that TraceSendData has only a single use, namely in the neighboring trace_flusher module; I'd agree with you if it had more uses. Anyway, this one is not important.